src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -141,12 +145,24 @@
 
           <!-- Actions Buttons -->
           <v-col v-if="!isSummaryView" class="pr-0">
-            <!-- orgPerson we have added/edited/removed: -->
-            <div
-              v-if="orgPerson.actions && orgPerson.actions.length > 0"
-              class="actions" :class="{'pr-5': wasRemoved(orgPerson)}"
-            >
-              <span v-if="wasAdded(orgPerson)" class="edit-action">
+            <!-- org-person that was replaced: -->
```

Comment:
I broke this up to make it easier to understand the various scenarios (which are now separate).